argilla-frontend/v1/domain/usecases/get-records-by-criteria-use-case.ts

```diff
@@ -97,7 +97,10 @@ export class GetRecordsByCriteriaUseCase {
           answer,
           suggestions,
           record.query_score,
-          recordPage
+          recordPage,
+          record.inserted_at,
```

Comment:
I read before that you named this argument as "metadataInsertedAt" but the parameter is named "record.inserted_at" so, I think does not make sense... right ?